client/rest-high-level/src/main/java/org/elasticsearch/client/dataframe/transforms/pivot/HistogramGroupSource.java

```diff
@@ -97,4 +100,38 @@ public boolean equals(Object other) {
     public int hashCode() {
         return Objects.hash(field, interval);
     }
+
+    public static Builder builder() {
```

Comment:
For some reason, I cannot leave comment in the line 55. Anyway, leaving it here: Please make the constructor package-private